charts/cryostat/templates/auth2_proxy.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-auth-proxy-secret
+data:
+  users.auth_proxy_htpasswd: {{ .Files.Get "templates/auth_proxy_htpasswd" | b64enc | quote }}
```

Comment:
https://helm.sh/docs/chart_template_guide/accessing_files/

> Some files cannot be accessed through the .Files object, usually for security reasons.
>    Files in templates/ cannot be accessed.

Reply:
Also,

```yaml
data:
  users.auth_proxy_htpasswd: something
```

is an unusual looking format for YAML - normally an object key would just be `users` and `auth_proxy_htpasswd` would be a key of a sub-object, or something like that. Maybe this also needs to be changed?

Reply:
> https://helm.sh/docs/chart_template_guide/accessing_files/
> 
> > Some files cannot be accessed through the .Files object, usually for security reasons.
> > Files in templates/ cannot be accessed.

Thank you for this!

Reply:
> Also,
> 
> ```yaml
> data:
>   users.auth_proxy_htpasswd: something
> ```
> 
> is an unusual looking format for YAML - normally an object key would just be `users` and `auth_proxy_htpasswd` would be a key of a sub-object, or something like that. Maybe this also needs to be changed?

It works the same with both `users.fileName: path` or just `fileName:path` the issue was accessing the file inside templates.